app/js/components/systemMetadata/systemMetadata.jade

```diff
@@ -1,11 +1,104 @@
-.row(ng-if='loading')
-  .col-sm-12
-    i.fa.spinner-xs.spinner
-    span Loading...
-.row.system-metadata(ng-if='!loading')
-  .col-sm-6(ng-repeat='facts in systemFacts')
-    table.meta-table.table-fixed
-      tbody
-        tr(ng-repeat='data in facts')
-          td(translate) {{data.label}}
-          td.metadata-value(translate) {{data.value}}
\ No newline at end of file
+.system-metadata(ng-if='!loading')
+  .flexgrid.equalheights
+    .row
+      // Hostname and UUID
+      .col-sm-7
+        .basic-information
+          .icon-inline
+            i.fa.fa-clone.fa-4x(ng-show="getSystemType() === 'Virtual'", tooltip="{{'Server type: Virtual' | translate}}")
+            i.fa.fa-server.fa-4x(ng-show="getSystemType() === 'Physical'", tooltip="{{'Server type: Physical' | translate}}")
+            i.fa.fa-question-circle.fa-4x(ng-show="getSystemType() === 'Unknown'", tooltip="{{'Server type: Unknown' | translate}}")
+
+            h2.hostname.ellipsis-overflow(translate, ng-if="system.hostname && !system.display_name") {{ system.hostname }}
+
+            // always show the uuid.... it must exist
+            .uuid.ellipsis-overflow
+              b UUID:&nbsp;
+              span(translate) {{ getUUID() }}
+            .alert-danger.text-center.pad5(ng-if='!system.isCheckingIn')
+              span(translate) This system is not checking in.
+
+      // Basic system meta-data table
+      .col-sm-5
+        table.meta-table
+          tbody
+            tr(ng-repeat='facts in initialMetadata')
+              td
+                b(translate) {{facts.label}}
+              td(translate) {{facts.value}}
+
+#expandableDataTable.expandable-data-table(ng-if='!loading; hasMetadata()', ng-class="{expanded: expandedTabs || expandIndefinite}")
+  // Toggle button
```

Comment:
Here are the states
- User hovers over an icon in the utility bar > the text name appears, this should be handled with css. We originally discussed setting a timeout on the hover, but it feels odd and adds unnecessary complexity.
- User hovers over the expand icon. The icon should have a hover state that is somewhere between the expanded and default states.
- The list is expanded > the expand icon should be styled (with higher specificity than the hover state) to be dark/reversed text. This is when the 'expanded' class should be added to the component.